.storybook/main.js

```diff
@@ -10,8 +10,6 @@ module.exports = {
   webpackFinal: async (config) => {
     return {
       ...config,
-      // Add our webpack plugins to default storybook plugins.
-      plugins: [...config.plugins, ...webpackConfig.plugins],
```

Comment:
MiniCssExtractPlugin was the only webpack plugin, so this can be removed without it